rest-api/src/main/resources/paths/index.yml

```diff
@@ -408,6 +412,8 @@
     $ref: ./users/v3_users.yml
 /v3/users/{userId}:
     $ref: ./users/v3_users_userId.yml
+/v3/users/self/dataSharing:
+    $ref: ./users/v3_users_self_dataSharing.yml
```

Comment:
I would prefer to just update the study participant's sharing status. I would like to remove these user APIs and replace them with unambiguously administrative or participant APIs, meaning /v3/participants or /v5/studies/{studyId}/participants for participants and /v1/organizations/{orgId}/members for administrative users (including Sage accounts). I have a fantasy that some day on the server, AccountService will be for admins and ParticipantService for participants without the two involving one another...

Reply:
In our discussion yesterday, we decided this is fine for now. At some point in the future, we might change setSharingStatus to internal only or something.